src/wallet/rpc/util.cpp

```diff
@@ -132,7 +132,10 @@ const LegacyScriptPubKeyMan& EnsureConstLegacyScriptPubKeyMan(const CWallet& wal
 
 std::string LabelFromValue(const UniValue& value)
 {
-    std::string label = value.get_str();
+    static const std::string empty_string;
+    if (value.isNull()) return empty_string;
```

Comment:
nit: not needed anymore now that we're returning a string copy

```suggestion
    if (value.isNull()) return "";
```

Reply:
Or `return {};`